design_pim/overview.rst

```diff
@@ -0,0 +1,327 @@
+How to customize any frontend part of the application
+-----------------------------------------------------
+
+Introduction
+===================
+
+Akeneo PIM uses requirejs and backbone to manage modules and views in our application. On top of those two layers, we added our own layer: the form extension system. In this documentation, we will discover the basics of those three layers to be able to understand Akeneo PIM stack and customize it.
```

Comment:
"RequireJS"
"Backbone"
(with capitals) or make it lowercase everywhere else for consistency.